app/containers/MessageComposer/components/Buttons/ActionsButton.tsx

```
@@ -9,6 +9,18 @@ import Navigation from '../../../../lib/navigation/appNavigation';
 import { useAppSelector, usePermissions } from '../../../../lib/hooks';
 import { useCanUploadFile, useChooseMedia } from '../../hooks';
 import { useRoomContext } from '../../../../views/RoomView/context';
+import { Platform, PermissionsAndroid, InteractionManager } from 'react-native';
```

ESLint

[error] `react-native` import should occur before import of `../../../../lib/database/services/Subscription`

(import/order)